### PR TITLE
Reservation approval

### PR DESCRIPTION
**Pull request by stephensxu**
- Host can approve or deny reservation requests
- Host can include a message in approval/denial
- Guest can view approved, denied and pending reservations
- Exact location of host and contact email for both host and guest will only be released after a reservation is approved

### CODE REVIEW

File: app/assets/stylesheets/main.css [hunk 1]
```diff
@@ -15,6 +16,8 @@ nav {
   float: left;
   color: green;
   font-weight: bold;
+  margin-bottom: 3em;
+  float: center;
 }
```
Comment (jfarmer) [inline app/assets/stylesheets/main.css:26]: There's no such thing as "float center"

File: app/controllers/kitchens_controller.rb [hunk 2]
```diff
@@ -16,19 +16,19 @@ def index
...
-    render :kitchen_reservations
+    render :kitchen_reservations_approved
   end
 
   def reservations_denied
     @kitchen = Kitchen.find(params[:id])
     @reservations = @kitchen.reservations.denied
-    render :kitchen_reservations
+    render :kitchen_reservations_denied
   end
```
Comment (jfarmer) [inline app/controllers/kitchens_controller.rb:33]: Why not just support URLs like `/kitchen/:id/:status`?

File: app/views/kitchens/kitchen_reservations_approved.erb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+  <% if @reservations.empty? %>
+    There is nothing here.
+  <% end %>
+ 
+  <ul>
+  <% @reservations.each do |reservation| %>
+    <li>
+    <hr>
+      <strong>Status: </strong> <%= reservation.status %><br>
```
Comment (jfarmer) [inline app/views/kitchens/kitchen_reservations_approved.erb:16]: You're secretly marking this up as a `<dl>` tag.  But more importantly, you've got to learn what goes in HTML and what goes in CSS.

Read Shay Howe's book.

File: app/views/reservations/_new_reservation_form.html.erb [hunk 4]
```diff
@@ -12,8 +12,8 @@
   <% end %>
 
   <div class="field">
-    <%= f.label "Reservation message" %><br>
-    <%= f.text_area :message, :class => "text_box" %>
+    <%= f.label "Introduce yourself, so the host knows you will be respectful" %><br>
```
Comment (jfarmer) [inline app/views/reservations/_new_reservation_form.html.erb]: Look up how the label tag works and what HTML this is generating.  This usage doesn't make sense.

File: app/views/users/index.html.erb [hunk 5]
```diff
@@ -8,7 +8,7 @@
   <% @kitchens.each do |kitchen| %>
     <% next if kitchen.data_status != "active" %>
   <hr>
-  <div class="kitchen_list">
+  <div class="individule_kitchen">
```
Comment (jfarmer) [inline app/views/users/index.html.erb]: individual

File: app/controllers/kitchens_controller.rb (hunk 2 above)
Comment (stephensxu) [inline app/controllers/kitchens_controller.rb:33]: how do I achieve that? Like by adding member routes to reservations?

File: app/views/users/index.html.erb (hunk 5 above)
Comment (stephensxu) [inline app/views/users/index.html.erb]: nice catch! haha

**Pull request merged**